src/test/datascience/editor-integration/codeGenerator.unit.test.ts

```diff
@@ -84,6 +84,51 @@ suite('Code Generator Unit Tests', () => {
         assert.equal(generatedCodes[0].generatedCodes[0].executionCount, 1, 'Wrong execution count');
     });
 
+    test('Execute %%latex magic in a cell with a cell marker', async () => {
```

Comment:
ha, I was about to mention that this change was a good candidate for unit tests

Reply:
Had to add an integration test as this was a bad break, basically as bad as not being able to run a cell.
To me this is a golden scenario, not being able to run latext cells in IW.